lib/resque/base_job.rb

```
@@ -32,10 +32,12 @@ def self.run_action(options, queue_name, action)
             retry
           else
             raise e
-          end  
+          end
         else
           raise e
         end
+      rescue => e
+        CartoDB.notify_exception(e)
```

Please raise the exception again.

By quickly looking into the code, this class has two purposes: retrying when there's a connection error and reporting resque stats properly. It was done by systems. In case of doubt ask systems.

By catching the exception and not re-throwing you're modifying the behavior